app/core/sale.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+export const validateMintTokensInputs = (
+  neoToMint: number,
+  gasToMint: number,
```

Comment:
These would need to be `string` instead of `number` if we want to go with the `toBigNumber` approach here.